### DISCUSSION TITLE
Contributor Call: 15 October 2022

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#2932] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @kaermorhan.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

**Details**
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: 15 October 2022 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/oct-15-2022/4pm).
Or check the bot in Discord! `#meetings`
The meetings will always be at 6pm, Antwerp time.

### Agenda
- Strapi: is it worth it?
- v3 docs: why *shouldn't* we publish them?
- Why creating new designs in the standalone environment is a safer bet

> Notes will be added after the call.

**Comment by l3xander** [comment, marked as answer]
Quick notes of this meeting 😅

- We decided not to migrate to Strapi v4, because the advantages don't weigh out the pain of migrating. We may look for something else, but we should be good in the meantime, since markdown directly doesn't seem to be intimidating to people.
- V3 docs are ready and @joostdecock wants to publish them already. No objections, so it's gonna happen.
- @/solstice from Discord: part of the gitpod documentation can be unclear to people unfamiliar with git/github, but @joostdecock is hesitant to add documentation as you can spiral into having to explain a lot and it's a lot to maintain. But it's important to find a good balance so we may come back to this.
- Use the standalone environment for pattern creation to avoid nightmareish pull requests. In a standalone environment it will set everything up for your own contained place. If you work from the monorepo/fork of monorepo, you have a shared history, and if you lag behind, it becomes harder to merge. 
Except if you work on a pattern that extends from another pattern, then working from the monorepo is better so you can change the pattern you're extending too.

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
